### PR TITLE
Add documentation for Chocolatey GUI 0.20.0 release

### PR DESCRIPTION
**Pull request by gep13**
## Description Of Changes

Information about the new features within Chocolatey GUI 0.20.0 have been added.

Screenshots have been updated, and missing sections of docs have been created.

## Motivation and Context

Without this information, folks won't know what has been included in the release.

## Testing

1. Ran the site locally, and made sure that all the pages looked as expected

## Change Types Made

* [ ] Bug fix (non-breaking change)
* [ ] Feature / Enhancement (non-breaking change)
* [ ] Breaking change (fix or feature that could cause existing functionality to change)
* [ ] PowerShell code changes.
* [x] Documentation changes

## Related Issue

N/A

## Change Checklist

* [x] Requires a change to the documentation
* [x] Documentation has been updated
* [ ] Tests to cover my changes, have been added
* [ ] All new and existing tests passed.
* [ ] PowerShell v2 compatibility checked.

### CODE REVIEW

File: input/en-us/chocolatey-gui/setup/configuration/settings/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 Order: 10
-xref: gui-settings
+xref: gui-configuration-settings
```
Comment (AdmiringWorm) [inline input/en-us/chocolatey-gui/setup/configuration/settings/index.md:3]: Will changing the Xref here also change the URL to the page?

If so, there may be a need to do a redirection.

File: input/en-us/chocolatey-gui/user-interface/main-window/package-details/index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+After either double clicking on a package, or choosing details from teh context menu, from the [main window](xref:gui-main-window),
+the details for the selected package will be shown.  This includes information such as:
+
+- Package Id
+- Version
+- Last update
+- External links provided by the package maintainer
+- Description
+- Release notes
```
Comment (AdmiringWorm) [inline input/en-us/chocolatey-gui/user-interface/main-window/package-details/index.md:16]: Not sure if we want to list all available information that can be displayed.

Some of the missing are

- Software Authors
- Package Maintainers (If available)
- Downloads for the version, and total downloads
- Size of the package
- Package Dependencies

File: input/en-us/chocolatey-gui/user-interface/settings/chocolatey-gui.md [hunk 3]
```diff
@@ -10,6 +10,9 @@ supports, as well as to set values for the specific configuration values that ex
 once changed, the feature or configuration value changes takes immediate effect, but it may require an application
 restart to take effect.
 
+There are search boxes which allows you to type the name of a feature, or a setting, and the list will be filtered down
+to only matching entries.
```
Comment (AdmiringWorm) [inline input/en-us/chocolatey-gui/user-interface/settings/chocolatey-gui.md:14]:
```suggestion
There are search boxes that allow you to type the name of a feature, or a setting, and the list will be filtered down
to only match entries with the search phrase in either the display name or the configuration name.
```

**Review by AdmiringWorm — DISMISSED** [review]
Just a few suggestions/comments, otherwise this looks good to me. I can't think of anything you missed either.

**Comment by st3phhays** [comment]
@gep13 I added a commit onto this which fixes the navigation.

Screen Shot 2022-02-08 at 11 54 37 AM

(Also, rebased this onto master during the process, and now it shows me as a coauthor of all the commits. Not too sure what I did there or if it's a problem. Maybe there was a better way I should have done that. )

**Comment by AdmiringWorm** [comment]
> (Also, rebased this onto master during the process, and now it shows me as a coauthor of all the commits. Not too sure what I did there or if it's a problem. Maybe there was a better way I should have done that. )

That is pretty much just what happens when you rebase a different persons commit.
@gep13 is still considered the author of the commits, but you are considered the committer since each commit would be updated (so the author haven't changed, but the one that committed each one changed).

**Comment by gep13** [comment]
@st3phhays yip, as @AdmiringWorm mentioned, this is totally expected, and not a problem.  Thanks for getting that change made, really appreciate it!

File: input/en-us/chocolatey-gui/setup/configuration/settings/index.md (hunk 1 above)
Comment (gep13) [reply input/en-us/chocolatey-gui/setup/configuration/settings/index.md:3]: No, there is no change required to the URL. That is actually one of the benefits of using xref's. The structure, url, etc of a page can change, but linking to an xref will always work. Far fewer changes needed when moving things around.

Comment (gep13) [reply input/en-us/chocolatey-gui/setup/configuration/settings/index.md:3]: In this scenario though, this one doesn't need changed, as there is already a page that has this xref, and they need to be unique.

**Pull request merged**